firefox-ios/Client/Frontend/Browser/ToastType.swift

```diff
@@ -5,19 +5,19 @@
 import Foundation
 
 enum ToastType: Equatable {
-    case singleTab
-    case allTabs(count: Int)
-    case singleInactiveTabs
-    case allInactiveTabs(count: Int)
+    case closedSingleTab
+    case closedSingleInactiveTab
+    case closedAllTabs(count: Int)
+    case closedAllInactiveTabs(count: Int)
```

Comment:
(Note: I renamed these for clarity but they're not needed for the fix. Can revert this if anyone thinks we should keep the previous names.)